Addimission_trial_code.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+      <h1 id="title">Survey Form</h1>
+  <br>
+      <p id="description">This is a survey form</p>
+<br>
```

Comment:
I  would suggest you fix the indentation with the br tag in this way you didn't face any difficulty during the checking.
Good Luck :+1: